src/index.js

```diff
@@ -22,4 +43,34 @@ function renderTasks() {
   });
 }
 
+function addTask(description) {
+  const task = {
+    description,
+    completed: false,
+    index: tasks.length + 1,
+  };
+
+  tasks.push(task);
+  saveTasks();
+  renderTasks();
+}
+
+const newTaskForm = document.querySelector('form');
+newTaskForm.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const newTaskInput = newTaskForm.querySelector('.new-task-input');
+  addTask(newTaskInput.value);
+  newTaskInput.value = '';
+});
+
```

Comment:
- [ ] Please add a function for deleting a task (remove an element from the array) . You can implement the deleting feature like [this](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/sneak_peek.md#:~:text=Your%20task%20is%20to%20build%20a%20website%20that%20displays%20a%20list%20that%20looks%20and%20behaves%20like%20the%20part%20of%20minimalist%20project%20capture%20in%20the%20video%20below.))

_Note: Removing all completed tasks and deleting a single task are both different features. You need to implement a function for deleting a single task_

Reply:
well resolved! ✔️